x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/index/fielddata/GeoShapeValues.java

```diff
@@ -70,6 +70,7 @@ public GeoShapeValue() {
             this.tile2DVisitor = new Tile2DVisitor();
         }
 
+        @SuppressWarnings("this-escape")
```

Comment:
same here?

Reply:
Again, this is something that I fixed in main but is breaking my local build. For sanity, I fixed it here. I will be a no-op once main is merged here